lessons/javascript-fundamentals/objects-and-arrays/README.md

```diff
@@ -0,0 +1,47 @@
+- title: JavaScript Fundamentals: Intro to Objects
+- tags: javascript, objects
+
+# Objectives
+* Be able to define basic objects and access properties
+* Explain the difference between objects and arrays
+* Be able to use slice, push, and pop methods to manipulate arrays 
+
+# Resources
+- [W3 JavaScript Intro to Objects](http://www.w3schools.com/js/js_objects.asp)
+- [W3 JavaScript Array Methods](http://www.w3schools.com/jsref/jsref_obj_array.asp)
+- [Working With Objects](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Working_with_Objects)
+
+#Lecture
+##Into to objects
+Objects are essentially lists of key/value pairs. 
+
+Objects are similar to arrays, except each value in an object has an explicit 'key', instead of an index. 
+
+We can access object values by their keys by using either 'dot' notation or brackets.
```

Comment:
Do we have some sweet real world examples? Ones I like:
- Mapping email addresses to passwords, for a login lookup
- Names mapped to favorite X (we can have a `favorite_foods` object, `favorite_movie`, etc)
- Mapping a company name to its RL address
etc.